content/en/docs/concepts/security/api-server-bypass-risks.md

```diff
@@ -59,7 +59,7 @@ API server. However, the Pod still runs on the node. For more information, refer
 ## The kubelet API {#kubelet-api}
 
 The kubelet provides an HTTP API that is typically exposed on TCP port 10250 on cluster
-worker nodes. The API might also be exposed on control plane nodes depending on the Kubernetes
+nodes. The API might also be exposed on control planes depending on the Kubernetes
```

Comment:
/hold

this is not accurate.

"control plane" in k8s is singular. it is a "plane" where the controller components reside with high availability and leader election in mind. that can be on a node or just a machine where no kubelet is run.
it depends on topology. some deployers / distributions of k8s like kubeadm and some proprietary too run a kubelet where the control plane components are, making it a "control plane node"